### PR TITLE
Fix price manipulation in oracle.

### PR DESCRIPTION
**Pull request by vivekvpandya**
## Issue
*Please replace this line with either a link to the ClickUp issue, or a reference to the GitHub issue.*

## Description
*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*

## Checklist

~- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_~
~- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_~

### CODE REVIEW

File: frame/oracle/src/tests.rs [hunk 1]
```diff
@@ -804,6 +804,109 @@ fn add_price() {
...
+        assert_ok!(Oracle::add_stake(Origin::signed(account_1), 50));
+        assert_ok!(Oracle::add_stake(Origin::signed(account_3), 50));
+        assert_ok!(Oracle::add_stake(Origin::signed(account_4), 50));
+        assert_ok!(Oracle::add_stake(Origin::signed(account_5), 50));
+
+        let balance1 = Balances::free_balance(account_1);
+        let balance3 = Balances::free_balance(account_3);
+        let balance4 = Balances::free_balance(account_4);
+        let balance5 = Balances::free_balance(account_5);
+        println!("BALANCES before price submissions:");
```
Comment (dzmitry-lahoda) [inline frame/oracle/src/tests.rs:858]: may be use log

File: frame/oracle/src/lib.rs [hunk 2]
```diff
@@ -804,7 +804,8 @@ pub mod pallet {
 			// because pre_prices.len() limited by u32
 			// (type of AssetsInfo::<T>::get(asset_id).max_answers).
 			if pre_prices.len() as u32 >= asset_info.min_answers {
-				if let Some(price) = Self::get_median_price(&pre_prices) {
+				// if let Some(price) = Self::get_median_price(&pre_prices) {
```
Comment (Dom-Roth) [inline frame/oracle/src/lib.rs:807]: Do we want to keep this comment?

File: frame/oracle/src/tests.rs [hunk 3]
```diff
@@ -856,6 +856,70 @@ fn submit_price_fails_stake_less_than_asset_slash() {
 	});
 }
 
+fn halborm_test_price_manipulation() {
```
Comment (Dom-Roth) [inline frame/oracle/src/tests.rs:859]: Should we also cover this test case with integration tests, to be safe?

Comment (vivekvpandya) [reply frame/oracle/src/tests.rs:859]: If it is not too hard in TS then sure I can try.

Comment (Dom-Roth) [reply frame/oracle/src/tests.rs:859]: Shouldn't be that hard, though if there's too much on your plate currently, the QA team can take care of this too. :)

Comment (vivekvpandya) [reply frame/oracle/src/tests.rs:859]: if QA team can do this then it will be great , I am working on other issues for halborn issue.

**Review by hussein-aitlahcen — APPROVED** [review]

File: frame/oracle/src/tests.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (vivekvpandya) [reply frame/oracle/src/tests.rs:859]: Name is descriptive enough.

**Pull request merged**